dockerfiles/docs.Dockerfile

```diff
@@ -16,8 +16,9 @@ COPY --from=hugo $GOPATH/bin/hugo /bin/hugo
 WORKDIR /src
 
 FROM build-base AS build
+ARG DOCS_BASEURL
 RUN --mount=type=bind,rw,source=docs,target=. \
-    hugo --gc --minify --destination /out
+    hugo --gc --minify --destination /out -b $DOCS_BASEURL
```

Comment:
what happens when building without passing a base-URL? will `-b` without value cause it to fail?

(what's the default if the option is not set? or should the ARG have a default (eg `localhost`) ?

Reply:
Can we please discuss this in https://github.com/distribution/distribution/pull/4102. I'd like @dvdksn to close this PR as the changes have been move to ☝️